luminoth/models/ssd/base_config.yml

```diff
@@ -0,0 +1,148 @@
+train:
```

Comment:
We shouldn't have a different `base_config.yml` for Faster and SSD, when most of it is repeated.